usecases/inboxes/inbox_users.go

```diff
@@ -136,11 +136,11 @@ func (usecase *InboxUsers) CreateInboxUser(ctx context.Context, input models.Cre
 }
 
 func (usecase *InboxUsers) UpdateInboxUser(ctx context.Context, inboxUserId string, role models.InboxUserRole) (models.InboxUser, error) {
-	inboxUser, err := transaction.TransactionReturnValue_deprec(
+	inboxUser, err := executor_factory.TransactionReturnValue(
```

Comment:
It looks weird to me.

Either we have utility function that take `usecase.TransactionFactory` as a param like this, or we have method lile `usecase.TransactionFactory.Transaction(...)` (cf `DeleteInboxUser` implementation)